universal-application-tool-0.0.1/app/views/questiontypes/RepeaterQuestionRenderer.java

```diff
@@ -36,43 +40,68 @@ public RepeaterQuestionRenderer(ApplicantQuestion question) {
 
   @Override
   public Tag render(Messages messages) {
-    ContainerTag repeaterFields = div().withId(REPEATER_FIELDS_ID);
-    // TODO: add each answer as a repeaterField
-    repeaterFields.with(repeaterField(PLACEHOLDER, Optional.empty(), 0));
+    RepeaterQuestion enumeratorQuestion = question.createEnumeratorQuestion();
+    ImmutableList<String> entityNames = enumeratorQuestion.getEntityNames();
+
+    ContainerTag enumeratorFields = div().withId(ENUMERATOR_FIELDS_ID);
+    int index;
```

Comment:
super nit: any reason why index can't be declared inside the for loop? I don't see it used afterwards

Reply:
it can be! originally this index was used to create new enumerator fields but that's changed now.